sdk/include/opentelemetry/sdk/trace/recordable.h

```diff
@@ -33,6 +34,14 @@ class Recordable
                       opentelemetry::trace::SpanId span_id,
                       opentelemetry::trace::SpanId parent_span_id) noexcept = 0;
 
+  /**
+   * Add an attribute to a span.
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-attributes

Perhaps rephrase "Add an attribute" to "Set an attribute"?

Reply:
Done.